domain/cloud/state/types.go

```diff
@@ -10,6 +10,8 @@ type CloudType struct {
 	Type string `db:"type"`
 }
 
+type AuthTypes []AuthType
```

Comment:
Comment please.

Reply:
all of these type definitions (in all domain state packages) have a global comment at the top of the file